src/lib/sticky-header/sticky-header.md

```diff
@@ -0,0 +1,48 @@
+Angular Material `<sticky-header>` is an attribute/component which makes DOM elements sticky to a specific parent.
+
+<!-- example(sticky-header-overview) -->
+
+The sticky-header is a useful component when users want to implement a container focusing on displaying headers,
```

Comment:
We don't need to worry about explaining _what_ the component is, since this is a pretty common concept. After the example, we can focus on how you set it up. Something like
```markdown
The parent scrolling container for a sticky-header must be marked with `cdkScrollable`.
This scrolling container contians one or more elements with `cdkStickyRegion`,
each with their own `cdkStickyHeader`. As the regions are scrolled, the header for the
region currently within the scrolling viewport will stick to the top.

DIAGRAM
```

What do you think of having a diagram like this?
https://docs.google.com/drawings/d/1XnDynnAyomqVFJDMDVIKmYiadRsTU2iyD0qPkDs1ark/edit